risk-map/yaml/controls.yaml

```
@@ -224,6 +315,26 @@ controls:
       - SDD
       - ISD
       - IMO
+    mappings:
+      mitre-atlas:
+        - AML.M0016
```

I think this is the wrong item... M0016 is `Vulneratbility Scanning` https://atlas.mitre.org/mitigations/AML.M0016.  Proposal:
- `Model Hardening` https://atlas.mitre.org/mitigations/AML.M0003
- `Use Ensemble Methods` https://atlas.mitre.org/mitigations/AML.M0006
- `Validate AI Model` https://atlas.mitre.org/mitigations/AML.M0008

```suggestion
        - AML.M0003
        - AML.M0006
        - AML.M0008
```